client/my-sites/plugins/eligibility-warnings/index.jsx

```diff
@@ -0,0 +1,142 @@
+			widgetWarnings,
+		} = this.props;
+
+		return (
+			<div className="eligibility-warnings">
+				<SectionHeader label={ translate( 'Conflicts' ) } />
+
+				{ errors.map( ( message ) =>
+					<Card key={ message.title } className="eligibility-warnings__message">
+						<Gridicon icon="notice" className="eligibility-warnings__error-icon" />
+						<div className="eligibility-warnings__message-content">
+							<div className="eligibility-warnings__message-title">
+								{ translate( 'Error' ) + `: ${ message.title }` }
+							</div>
+							<div className="eligibility-warnings__message-description">
+								{ message.description }
+							</div>
+						</div>
+						<div className="eligibility-warnings__message-action">
+							<Button href={ message.actionUrl } target="_blank" rel="noopener noreferrer">
+								{ translate( 'Resolve' ) }
+							</Button>
+						</div>
+					</Card>
+				) }
+
+				{ pluginWarnings.map( ( message ) =>
+					<Card key={ message.title } className="eligibility-warnings__message">
+						<Gridicon icon="notice" className="eligibility-warnings__warning-icon" />
+						<div className="eligibility-warnings__message-content">
+							<div className="eligibility-warnings__message-title">
+								{ translate( 'Unsupported feature' ) + `: ${ message.title }` }
+							</div>
+							<div className="eligibility-warnings__message-description">
+								{ message.description }
+							</div>
+						</div>
+						<div className="eligibility-warnings__message-action">
+							<a href={ message.actionUrl } target="_blank" rel="noopener noreferrer">
+								<Gridicon icon="help-outline" className="eligibility-warnings__warning-action" />
+							</a>
+						</div>
+					</Card>
+				) }
+
+				{ widgetWarnings.map( ( message ) =>
+					<Card key={ message.title } className="eligibility-warnings__message">
+						<Gridicon icon="notice" className="eligibility-warnings__warning-icon" />
+						<div className="eligibility-warnings__message-content">
+							<div className="eligibility-warnings__message-title">
+								{ translate( 'Incompatible widget' ) + `: ${ message.title }` }
+							</div>
+							<div className="eligibility-warnings__message-description">
+								{ message.description }
+							</div>
+						</div>
+						<div className="eligibility-warnings__message-action">
+							<a href={ message.actionUrl } target="_blank" rel="noopener noreferrer">
+								<Gridicon icon="help-outline" className="eligibility-warnings__warning-action" />
+							</a>
+						</div>
+					</Card>
+				) }
+
+				<Card className="eligibility-warnings__confirm-box">
+					<Gridicon icon="info-outline" className="eligibility-warnings__confirm-icon" />
+					<div className="eligibility-warnings__confirm-text">
+						{ ! isEligible && translate(
+							'You must resolve the errors above before proceeding. '
+						) }
+						{ isEligible && translate(
+							'By proceeding you understand that you will loose these features. '
```

Comment:
What if we said it more like:

`If you proceed you will no longer be able to use these features.`